src/pages/rockets/Rockets.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Loader from '../../components/Loader';
+import {
+  fetchRockets,
+  leaveRockets,
+  reserveRockets,
+} from '../../redux/feature/rockets/rocketsSlice';
+import style from './Rockets.module.css';
+
+const Rockets = () => {
+  const { rockets, loading, fetched } = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+  const ReserveRocket = (id) => dispatch(reserveRockets(id));
+  const leaveRocket = (id) => dispatch(leaveRockets(id));
+  console.log(rockets);
+  useEffect(() => {
+    if (!fetched) {
```

Comment:
every time a page loads it is better if you fetch data to get recent records unless you know the data is static and changes occasionally.